### PR TITLE
Opcode pricing with policy contract update

### LINKED ISSUE #312: Adapt to adjustable opcode pricing
With neo3-preview4 opcodes are not statically priced anymore. Rather, they receive a factor which is multiplied by a base execution fee resulting in the opcode price - the reference opcode for the base fee is NOP.
We need to adapt the neow3j SDK to reflect those changes. This mostly affects the fee calculations for transactions.  
Check out the method  `Wallet.CalculateNetworkFee(...)` in the neo-core (i.e., neo-project/neo). There's the code that calculates the network fee.

See:
- https://github.com/neo-project/neo/issues/1875
- https://github.com/neo-project/neo/pull/2045.

### PR DESCRIPTION
**Pull request by mialbu**
Closes #312, #315

### CODE REVIEW

File: contract/src/test/java/io/neow3j/contract/PolicyContractTest.java [hunk 1]
```diff
@@ -6,8 +6,7 @@
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.*;
```
Comment (gsmachado) [inline contract/src/test/java/io/neow3j/contract/PolicyContractTest.java:9]: wildcard?! noooooooooooooooooooooooo

🤣🤣🤣🤣🤣🤣🤣🤣🤣

Comment (mialbu) [reply contract/src/test/java/io/neow3j/contract/PolicyContractTest.java:9]: I set up my laptop from scratch and forgot to disable wildcards in the my IDE when I installed it again...

**Comment by mialbu** [comment]
Due to dependencies of those changes in other preview4 changes, I will merge this pr into `feature/sdk-preview4`. The changes will then be handled in that branch.

**Comment by gsmachado** [comment]
@mialbu great. Can we delete this branch, then? 👍

**Pull request merged**